e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -47,6 +47,9 @@
               },
               "transportType": {
                 "value": "amqp"
+              },
+              "outputName": {
```

Comment:
>outputName [](start = 15, length = 10)

Is the output configurable?

Reply:
https://github.com/Azure/iotedge/blob/master/edge-modules/load-gen/Program.cs#L81
Yes it seems like it is